### ISSUE TITLE
Promoting player to main tank

### ISSUE THREAD

**Issue by Schm0ftie** [waha/m_UI#4]
When trying to promote a player to a main tank I get always the message:

"m_UI has been blocked from an action only available to the Blizzard UI. You can
disable this addon and reload the UI."

I have tried to solve the problem my self but I didnt fint anything.

**Comment by waha** [comment]
I just took a look at it, and it's the armory link dropdown menu feature that taints. Not sure if this even can be fixed since we're injecting insecure code thus automatically tainting.  I'd say just go with armory_link = false in your cfg.lua for now.  Hadn't spotted this as I'm never raid leading.

**Comment by Schm0ftie** [comment]
Well, the armory link is a nice feature but I give ot up anytime to be able to set main tanks.
Thanks a lot

**Comment by waha** [comment]
Indeed, it's a nice feature.  I've been looking into this a bit more and as far as I can tell there is no way to add menu items to the right click menu without causing taints.  So I think the most viable solution will be to simply remove the armory link feature.

**Comment by Schm0ftie** [comment]
Yeah I think it's the right call to remove it. If needed you could simply modulize it in a single addon and change it so that it works in a way that does not use the rightclick and so doesn't taint. Don't think many are using that feature.
